styles/elk/catppuccin.user.css

```
@@ -9,12 +9,16 @@
 @license 		MIT
 
 @preprocessor less
-@var select flavor "Flavor" ["latte:Latte", "frappe:Frappe", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select lightFlavor "Light Flavor" ["latte:Latte", "frappe:Frappe", "macchiato:Macchiato", "mocha:Mocha*"]
```

```suggestion
@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappe", "macchiato:Macchiato", "mocha:Mocha"]
```
This just means that the default light mode flavour will be latte